externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala

```diff
@@ -72,7 +72,7 @@ case class EnginePage(parent: EngineTab) extends WebUIPage("") {
     }
       <li>
         <strong>Time since start: </strong>
-        {formatDurationVerbose(timeSinceStart)}
```

Comment:
It looks like the Spark thrift server's page is also using the `formatDurationVerbose` method.

https://github.com/apache/spark/blob/f0e18284d0c8938cb35f2ad3668aff0e4ef4891c/sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala#L65

Reply:
Yes, for `Time since start` , Spark uses `formatDurationVerbose`. But for the duration in the table, `formatDuration` is used.

Reply:
Reverted to use formatDurationVerbose for 'Time since start'.